src/main/cpp/apple_fsnotifier.cpp

```diff
@@ -198,7 +198,7 @@ Java_net_rubygrapefruit_platform_internal_jni_OsxFileEventFunctions_startWatchin
     for (int i = 0; i < count; i++) {
         jstring path = (jstring) env->GetObjectArrayElement(paths, i);
         char* watchedPath = java_to_char(env, path, result);
-        printf("~~~~ Watching %s\n", watchedPath);
+        log_info(env, "~~~~ Watching %s\n", watchedPath);
```

Comment:
I suppose we don't need the `~~~~` any more, right?

Reply:
I'll rewrite these soon, and we can change the log messages then.